api/src/pcapi/utils/sentry.py

```diff
@@ -11,11 +14,27 @@
 from pcapi.utils.health_checker import read_version_from_file
 
 
-def ignore_flask_shell_event(
-    event: dict[str, typing.Any], _hint: dict[str, typing.Any]
-) -> dict[str, typing.Any] | None:
+# Do not suppress warnings
+# https://docs.python.org/3/library/warnings.html#the-warnings-filter
+os.environ["PYTHONWARNINGS"] = "default"
+
+# Redirect all warnings to the logging library
+logging.captureWarnings(True)
```

Comment:
Est-il possible de formater l'event Sentry et ne conserver que `core/finance/repository.py:84: SAWarning` en titre, et le reste du message de log dans le corps ?

Reply:
Ça doit certainement être possible, et ça sera beaucoup plus lisible en effet, je vais regarder ça : )

Reply:
C'est fait ! https://sentry.passculture.team/organizations/sentry/issues/?environment=development&project=5&query=is%3Aunresolved&statsPeriod=24h